src/App.vue

```diff
@@ -113,7 +113,7 @@ export default {
 			return this.backgroundFetching
 				&& this.webNotificationsGranted
 				&& this.userStatus !== 'dnd'
-				&& this.tabId !== this.lastTabId
+				&& this.tabId === this.lastTabId
```

Comment:
due to `this.lastTabId = response.lastTabId` in the second part of the diff in this file, we always compared `<tabid> !== undefined` which was always true and therefor multiple tabs gave multiple desktop notifications again.